libflux/flux-core/src/semantic/flatbuffers/mod.rs

```diff
@@ -1104,6 +1106,71 @@ impl<'a, 'b> SerializingVisitorState<'a, 'b> {
             },
         ))
     }
+
+    fn create_function_expression(&mut self, func: &semantic::nodes::FunctionExpr) -> WIPOffset<fbsemantic::FunctionExpression<'a>> {
```

Comment:
Is this code a copy from above? I think it should be possible to do this without duplicating code. 

I think what needs to happen is that the visitor that walks over the semantic graph needs to be updated to visit the `vectorized` field of a function expression. If you do that then I would expect that the expression would be on the `expr_stack`.